src/index.html

```diff
@@ -5,8 +5,388 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="nav">
+        <a href="link" class="nav__logo">
+          <img src="images/logo.png" alt="logo">
+        </a>
+
+        <nav class="nav__blocks">
+          <div class="nav__links">
```

Comment:
nav__item would be a better name than nav__links. nav__links sounds plural, although it's a div with 1 link inside of it